api/internal/core/store/validate.go

```diff
@@ -230,14 +231,21 @@ func (v *APISIXJsonSchemaValidator) Validate(obj interface{}) error {
 	}
 
 	//check plugin json schema
-	plugins := getPlugins(obj)
+	plugins, schemaType := getPlugins(obj)
+	//fix lua json.encode transform lua{properties={}} to json{"properties":[]}
```

Comment:
we can avoid this hack, Lua `json.encode` can do this in a good way.

@nic-chen here is the API, take a look: https://github.com/openresty/lua-cjson/#encode_empty_table_as_object

you can help @liuxiran to fix this